api/requirements-dev.txt

```
@@ -79,11 +79,11 @@ six==1.16.0
     #   requests-mock
 tox==4.11.3
     # via -r requirements-dev-minimal.txt
-urllib3==2.0.7
+urllib3==2.1.0
```

Downgrade urllib3 to meet the requirements of botocore.

```suggestion
urllib3==2.0.7
```